### PR TITLE
Updated exchanges.html to add Australian and New Zealand exchange

### PR DESCRIPTION
**Pull request by stefanidi**
Independent Reserve has been operating since 2013 and is one of the leading exchanges in Australia and New Zealand

### CODE REVIEW

File: _templates/exchanges.html [hunk 1]
```diff
@@ -168,7 +169,8 @@ <h3 id="nepal"><img src="/img/flags/NP.png" alt="Nepalese flag">Nepal</h3>
     <div>
       <h3 id="new-zealand"><img src="/img/flags/NZ.png" alt="New Zealand Flag">New Zealand</h3>
       <p>
-        <a href="https://kiwi-coin.com/">Kiwi-coin</a>
+        <a href="https://kiwi-coin.com/">Kiwi-coin</a><br>
+        <a href="https://www.independentreserve.com/">Independent Reserve</a>
```
Comment (wbnns) [inline _templates/exchanges.html]: We list the exchanges alphabetically under each country - could you please fix this placement, here?

**Comment by seandotau** [comment]
Independent Reserve is an Australian company and operates in Australia. It accepts deposits in AUD, NZD and USD but doesn't actually operate in NZ.

**Comment by stefanidi** [comment]
The requested change has been done.

**Comment by wbnns** [comment]
@stefanidi Great, thank you.

**Pull request merged**